test/server/data/expected-test-run-errors/action-invalid-object-property-error

```diff
@@ -0,0 +1,21 @@
+The "ClickOptions" object contains an invalid property: "invalidProp". You can safely remove it due to this property is not applied.
+Make sure you have correctly specified the action options. List of available options for the "ClickOptions" are:
+"ctrl" and "alt".
```

Comment:
I propose changing the display format of valid properties for better reading - they should be ordered by name and rendered in a column:
```js
The "ClickOptions" object contains an invalid property: "invalidProp". You can safely remove it due to this property is not applied.
Make sure you have correctly specified the action options. 
List of available options for the "ClickOptions" are:
"alt",
"ctrl".
```